packages/controller/src/types.ts

```diff
@@ -183,6 +183,8 @@ export type ProfileOptions = IFrameOptions & {
   namespace?: string;
   /** The tokens to be listed on Inventory modal */
   tokens?: Tokens;
+  /** The policies to use for the profile */
+  policies?: SessionPolicies;
```

Comment:
Is this intended to be part of this PR?

Reply:
Yes it does because we need to know the entrypoints defined in the policies inside the profile iframe to then query the corresponding transactions from toriis